docker/sql/schema.sql

```diff
@@ -52,3 +52,38 @@ CREATE TABLE aws_bill_repository (
 CREATE VIEW aws_bill_repository_due_update AS
 	SELECT * FROM aws_bill_repository WHERE next_update <= NOW()
 ;
+
+CREATE TABLE aws_fetch_pricing (
+	id INTEGER NOT NULL AUTO_INCREMENT,
+	product VARCHAR(255) NOT NULL,
+	etag VARCHAR(255) NOT NULL,
+	CONSTRAINT PRIMARY KEY (id),
+	CONSTRAINT UNIQUE KEY (product)
+);
+
+CREATE TABLE aws_region (
```

Comment:
Because you just store the same data in `region` and `pretty`, this table adds nothing and increases complexity. I think you should have a look at “Codd’s normal forms”, BTW.